scripts/globals/effects/subtle_sorcery.lua

```diff
@@ -14,14 +14,12 @@ effect_object.onEffectGain = function(target, effect)
 end
 
 effect_object.onEffectTick = function(target, effect)
-    target:addMod(xi.mod.ENMITY, -20)
 end
 
 effect_object.onEffectLose = function(target, effect)
     local jpValue = target:getJobPointLevel(xi.jp.SUBTLE_SORCERY_EFFECT)
 
     target:delMod(xi.mod.MACC, 100)
-    target:delMod(xi.mod.ENMITY)
```

Comment:
Throws an error because no value is specified.

Reply:
This the addMod was removed, should have no impact from removing this value here.